data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/HasTagsExpressionFunction.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import org.opensearch.dataprepper.model.event.Event;
+import java.util.stream.Collectors;
+import java.util.List;
+import javax.inject.Named;
+import java.util.function.Function;
+
+@Named
+public class HasTagsExpressionFunction implements ExpressionFunction {
```

Comment:
You will need to rebase this after the PR for the hasTags is merged

Reply:
@kkondaka , You can submit PRs as draft PRs if you are including work from another PR that is in review.